server/src/main/scala/de/frosner/broccoli/templates/TemplateRenderer.scala

```diff
@@ -7,12 +7,25 @@ import de.frosner.broccoli.models.{Instance, ParameterInfo, ParameterType}
 import org.apache.commons.lang3.StringEscapeUtils
 import play.api.libs.json.{JsString, JsValue, Json}
 
+import scala.util.Try
+
 /**
   * Renders json representation of the passed instance
   *
   * @param defaultType The default type of template parameters
   */
 class TemplateRenderer(defaultType: ParameterType) {
+
+  /** Checks if the value is a number or a is a number **/
+  def isNumber(value: String): Boolean = {
```

Comment:
I think the safest and best way to check if something can be a number is to try to convert it. Everything else usually has corner cases you didn't think about.

IMO we should check what datatype the JSON number from play expects (I believe it might be BigDecimal or Double), try to parse the string in a `Try` and return that.

Reply:
Had a discussion with Aleks and I am moving away from regex.